Source/SwiftLintCore/Extensions/Array+SwiftLint.swift

```diff
@@ -62,6 +62,46 @@ public extension Array {
         Dictionary(grouping: self, by: { transform($0) })
     }
 
+    /// Group the elements in this array into a dictionary, keyed by applying the specified `transform`.
+    /// Elements for which the `transform` returns a `nil` key are removed.
+    ///
+    /// - parameter transform: The transformation function to extract an element to its group key,
+    ///                        or exclude the element..
```

Comment:
```suggestion
    ///                        or exclude the element.
```

Same in the other method.